contrib/system_containers/rhel/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM rhel7:7-released
```

Comment:
Why were the OIT headers removed?  I assume OIT will still be responsible for the rhel builds.  Yes?

Reply:
I can bring them in again, I removed them mainly cause it's RH specific

Reply:
Put the headers back, I just removed references to internal repos and IRC channels but the #oit headeres are back now. Thanks Jhon.

Reply:
The oit headers are only intended to be "seen" in distgit. They are added dynamically when moving the file from this upstream repo to the downstream distgit. You can make notes in this file documenting how things work for your own purposes, but the oit comments do not need to be preserved.